.gitignore

```diff
@@ -1 +1,3 @@
-.idea/
\ No newline at end of file
+.idea/
+__pycache__/
+textract_test.py
```

Comment:
Why not include tests?

Reply:
I can absolutely include it if you want. That script specifically, checks the speed of pdf extraction packages and has little to do with the intended use of the repo.